src/util/validationUtils.js

```diff
@@ -169,9 +169,30 @@ const checkNoUnsupportedParams = (query, unsupportedParams, operationName) => {
   }
 };
 
+/**
+ * Pulls query parameters from both the body and creates a new parameters map
+ * @param {Object} query the query terms on the request URL
+ * @param {Object} body a FHIR Parameters object passed into the body of the request passed in
```

Comment:
I'm pretty sure we are bringing in the entire request body (which isn't really an issue because it's expected to just be a FHIR Parameters resource), so we should take note that this is technically the entire request body